src/storage/txn/commands/compare_and_swap.rs

```diff
@@ -218,15 +252,17 @@ mod tests {
     }
 
     fn test_cas_process_write_impl<F: KvFormat>() {
+        let ts_provider = if F::TAG == kvproto::kvrpcpb::ApiVersion::V2 {
```

Comment:
Suggest to extract this block to a utility function.

Reply:
fixed, thanks!